src/SFA.DAS.AssessorService.Web/Views/ManageUsers/Index.cshtml

```diff
@@ -31,29 +31,30 @@
                         </tr>
```

Comment:
Whilst not part of this PR, we're exposing and API Type directly to the view. This should be a View Model

Reply:
On the whole, I agree, but in this case the VM and the type would be identical and it would be a map between the two for no reason.  If that part of the view becomes more complicated and view specific, then yeah, VM all the way.